src/app/settings/settings.component.html

```diff
@@ -51,7 +51,14 @@
         </button>
         <div *ngIf="showSuccessMessage" style="color: green" class="fs-15px ml-10px">Success!</div>
       </div>
-
+      
+      <div class="fs-15px pt-30px">BitClout Theme</div>
```

Comment:
I think having it below works pretty good but maybe update the button text above to "Update Email" rather then "Update Settings"

Reply:
From a UX perspective, it doesn't make much sense. If this affordance is a "setting" in a "settings page" then it should be part of the page and updating the settings should all work the same way.

Just my two-cents as a UX designer.